net/redistest/redistest.go

```diff
@@ -15,6 +15,11 @@ func NewTestRedis(t testing.TB) (address string, done func()) {
 }
 
 func NewTestRedisWithPassword(t testing.TB, password string) (address string, done func()) {
+
+	if fixDeadline, _ := time.Parse("2006-01-02", "2023-08-01"); time.Now().Before(fixDeadline) {
```

Comment:
Could use `time.DateOnly` but its only available since go1.20

Reply:
I don't get the relevance of 2006 date